### ISSUE TITLE
Remove Manual librdkafka compile from Connect Dockerfile

### ISSUE THREAD

**Issue by dixonwhitmire** [LinuxForHealth/connect#189]
The current version of the connect Dockerfile manually compiles the librdkafka library, used by the confluent client, since the version we need, 1.6, is not in the stable Alpine repository - https://pkgs.alpinelinux.org/packages?name=librdkafka&branch=v3.13. 

The latest librdkafka release, 1.7 is available in the edge repository. 1.6 should be released by the end of May. Once 1.6 is available we can update our Dockerfile to just use "apk" to download the package.

**Comment by dixonwhitmire** [comment]
librdkafka is now available for alpine 3.14 - https://pkgs.alpinelinux.org/packages?name=librdkafka&branch=v3.14&repo=community

We will be able to move forward once our Python base image supports alpine 3.14 (currently it supports 3.13) - https://hub.docker.com/_/python

**Comment by dixonwhitmire** [comment]
A new image, python-3.9.6-alpine-3.14 is available on dockerhub. This image supports librdkafka 1.7

```shell
tdw@dixons-mbp ~ % docker run -ti --rm python:3.9.6-alpine3.14 ash
Unable to find image 'python:3.9.6-alpine3.14' locally
3.9.6-alpine3.14: Pulling from library/python
5843afab3874: Pull complete 
2dfaacf7024e: Pull complete 
2dbba127c6aa: Pull complete 
1a467efa2204: Pull complete 
728751b4406e: Pull complete 
Digest: sha256:3fac4a8ca1926c5a0b8fc7165079a58c7868f2b46e2acc263469b4c2fcab68fa
Status: Downloaded newer image for python:3.9.6-alpine3.14
/ # apk update && apk add librdkafka
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
v3.14.0-158-g620a4dae1c [https://dl-cdn.alpinelinux.org/alpine/v3.14/main]
v3.14.0-155-g6bc8b5c548 [https://dl-cdn.alpinelinux.org/alpine/v3.14/community]
OK: 14938 distinct packages available
(1/6) Installing libgcc (10.3.1_git20210424-r2)
(2/6) Installing lz4-libs (1.9.3-r0)
(3/6) Installing libsasl (2.1.27-r12)
(4/6) Installing libstdc++ (10.3.1_git20210424-r2)
(5/6) Installing zstd-libs (1.4.9-r1)
(6/6) Installing librdkafka (1.7.0-r0)
```
